examples/atmega2560p/blink/target/CACHEDIR.TAG

```diff
@@ -0,0 +1,3 @@
+Signature: 8a477f597d28d172789f06886806bc55
```

Comment:
Target folder should not be included, remove everything from that

Reply:
deleted from the repository